examples/plot_alignment_methods_benchmark.py

```diff
@@ -156,18 +179,61 @@
 
     # plot correlation for each method
     aligned_score = roi_masker.inverse_transform(method_error)
```

Comment:
I would remove this comment and move this line of code just under the next comment !